src/dialogs/more-info/controls/more-info-update.ts

```diff
@@ -257,6 +286,7 @@ class MoreInfoUpdate extends LitElement {
                   <ha-switch
                     slot="end"
                     id="create-backup"
+                    .checked=${this._createBackup}
```

Comment:
We should update `_createBackup` when the switch is changed. And then we can also use it for the rest of the logic when calling the update service.